### ISSUE TITLE
Feature Request: publicly accessible tags

### ISSUE THREAD

**Issue by PostPollux** [linuxserver/Heimdall#323]
Would be awesome if it were possible to decide if a tag is publicly accessible, or not, too. Right now, if I have a user with a password, i can only set the front page to be publicly accessible.
I think it would be good to add that option to tags also.

**Comment by derek-shnosh** [comment]
Agreed. I'm trying to find out how to remove the password from the admin account now because of this.

**Comment by PostPollux** [comment]
Yes, that's possible, but I explicitly want to have a password for the admin account. I thought of making heimdall the start page of our company and I would like to configure it in a way that only the administrator can change things, but that the front page plus the tags (which are only there for organization) are publicly acessible.

**Comment by derek-shnosh** [comment]
Same page here @PostPollux, I've just done this as an interim workaround since I'm restricting access to the host via other means.

However @KodeStar, I've found that setting the password to `null` doesn't really delete the password; i.e. when I accessed the page without login and clicked a tag, I was still prompted for a password. I had to type **null** to see the tag page.

**Comment by judexzhu** [comment]
I love the idea to public access the tags. It would help me to separate the same kind of apps by tag, and let people have more clear idea know which one of them they need.

For example, multiple prometheus with multiple grafanas

**Comment by dekoenpi** [comment]
https://github.com/linuxserver/Heimdall/issues/323#issuecomment-462033053
same here
